src/lib/state/DY.Lib.State.PKI.fst

```diff
@@ -60,10 +60,10 @@ val is_public_key_for:
 let is_public_key_for #cinvs tr pk pk_type who =
   match pk_type with
   | PkEnc usg -> (
-    is_encryption_key (PkKey usg empty) (principal_label who) tr pk
+    is_encryption_key (PkKey usg empty) (principal_tag_label who "DY.Lib.State.PrivateKeys") tr pk
```

Comment:
It feels like these label constructors should be defined somewhere in the library, so that if the constant string updates, it only needs to change in one place, and so that the constructor doesn't need to be redefined in examples (as it currently is)

Reply:
done